src/hotspot/cpu/aarch64/interpreterRT_aarch64.cpp

```diff
@@ -292,7 +385,8 @@ class SlowSignatureHandler
       *_int_args++ = from_obj;
       _num_int_args++;
     } else {
-      *_to++ = from_obj;
+      store_and_inc(_to, from_obj, NativeStack::intSpace);
+
       _num_int_args++;
```

Comment:
`pass_byte()` and  `pass_short()` use only one `_num_int_args++;` after the `if else` but other methods use 2 of them inside `if else` branches.

We should be consistent.

Reply:
Agree. I'm going to do as much refactoring as needed before this patch under JDK-8261071

Reply:
> I'm going to do as much refactoring as needed before this patch under JDK-8261071

The recent merge resolves inconsitency between pass_byte/pass_short and other methods.